booster/tools/booster/Proxy.hs

```diff
@@ -177,15 +186,19 @@ respondEither cfg@ProxyConfig{statsVar, boosterState} booster kore req = case re
                         logStats SimplifyM (boosterTime + koreTime, koreTime)
                         when (koreRes.state /= boosterRes.state) $ do
                             bState <- liftIO (MVar.readMVar boosterState)
-                            Log.logOtherNS "proxy" (Log.LevelOther "Aborts") $
-                                let m = fromMaybe bState.defaultMain simplifyReq._module
-                                    def =
-                                        fromMaybe (error $ "Module " <> show m <> " not found") $
-                                            Map.lookup m bState.definitions
-                                    diff =
-                                        fromMaybe "<syntactic difference only>" $
-                                            diffBy def boosterRes.state.term koreRes.state.term
-                                 in Text.pack ("Kore simplification: Diff (< before - > after)\n" <> diff)
+
+                            Booster.Log.withContext "proxy" $
+                                Booster.Log.withContext "abort" $
+                                    Booster.Log.withContext "detail" $
```

Comment:
we may need a different context here... maybe `diff`? perhaps best left for a future tweak